ui-components/src/forms/MultiSelectField.scss

```diff
@@ -0,0 +1,232 @@
+  box-shadow: none;
+}
+
+.multi-select-field__option:hover {
+  background: var(--bloom-color-blue-200);
+  color: var(--bloom-color-primary);
+  margin: 0;
+  cursor: pointer;
+}
+
+.multi-select-field__option--focused {
+  box-shadow: 0 0 2px 2px var(--bloom-color-accent-cool),
+    inset 0 0 3px var(--bloom-color-accent-cool) !important;
+}
+
+.multi-select-field__option:hover[aria-disabled="true"],
+.multi-select-field__option--focused[aria-disabled="true"] {
+  background: var(--bloom-color-gray-300);
+}
+
+.multi-select-field__option--no-results,
+.multi-select-field__option[aria-disabled="true"] {
+  color: var(--bloom-color-gray-750) !important;
+  cursor: not-allowed !important;
+  text-decoration: none !important;
+}
+
+.multi-select-field__option--no-results {
+  margin-top: 0 !important;
+  background: transparent !important;
+}
+
+.multi-select-field__selected {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  border: 1px solid var(--bloom-color-gray-500);
+  padding-block: var(--bloom-s3);
+  padding-inline: var(--bloom-s5) var(--bloom-s10);
+  margin: 2px 4px 2px 0;
+  font-size: var(--bloom-font-size-tiny);
+  border-radius: var(--bloom-rounded-full);
+  word-break: break-word;
+  cursor: pointer;
+}
+
+.multi-select-field__selected--disabled {
+  opacity: 0.7;
+}
+
+.multi-select-field__selected:hover {
+  background-color: var(--tag-hover-background-color);
+  border-color: var(--tag-hover-background-color);
+  color: var(--bloom-color-white);
+}
+
+.multi-select-field__selected:focus {
+  outline: none;
+  box-shadow: 0 0 2px 1px var(--bloom-color-accent-cool);
+}
+
+.multi-select-field__selected:after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  right: 12px;
+  width: 20px;
+  height: 20px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='10' height='10' viewBox='0 0 10 10' version='1.1'%3E%3Cg id='surface1'%3E%3Cpath d='M 0.234375 10 L 5 5.21875 L 9.765625 10 L 9.988281 9.78125 L 5.21875 5 L 9.988281 0.21875 L 9.765625 0 L 5 4.78125 L 0.234375 0 L 0.0117188 0.21875 L 4.78125 5 L 0.0117188 9.78125 Z M 0.234375 10 ' style='stroke:%230077da;fill-rule:nonzero;fill:%23ffffff;fill-opacity:1;'/%3E%3C/g%3E%3C/svg%3E");
+  transform: translateY(-50%);
+}
+
+.multi-select-field__selected:hover:after {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='10' height='10' viewBox='0 0 10 10' version='1.1'%3E%3Cg id='surface1'%3E%3Cpath d='M 0.234375 10 L 5 5.21875 L 9.765625 10 L 9.988281 9.78125 L 5.21875 5 L 9.988281 0.21875 L 9.765625 0 L 5 4.78125 L 0.234375 0 L 0.0117188 0.21875 L 4.78125 5 L 0.0117188 9.78125 Z M 0.234375 10 ' style='stroke:%23ffffff;fill-rule:nonzero;fill:%23ffffff;fill-opacity:1;'/%3E%3C/g%3E%3C/svg%3E");
+}
+
+@media speech {
```

Comment:
can you walk me through what the speech query does? not finding great resources online

Reply:
@emilyjablonski I have no idea TBH. 😅 Some of this stylesheet was pulled from the aria-autocomplete example.